demo/glfw_opengl2/Makefile

```diff
@@ -2,7 +2,7 @@
 BIN = demo
 
 # Flags
-CFLAGS += -std=c89 -Wall -Wextra -pedantic -O2
+CFLAGS += -std=c99 -Wall -Wextra -pedantic -O2
```

Comment:
We do target C89 across all the other demos. Mind if we keep this?

Reply:
Let me clarify that C89 is just a nice-to-have when it comes to code which won't end up in `nuklear.h`. If there are no other issues with this PR and if @crazyBaboon will not want to make it C98-compatible, we'll merge it.

Reply:
The problem are the C++ style comments in the stb_image.h.

Ideally, these should be replaced by C style comments, or else it won't compile (with the C89 flag) using some mainstream compilers. But that is a PITA. 

Does anyone know if there is a way of using C++ comments with the C89 flag?

Reply:
Awesome! Sounds good with me :+1:

Reply:
I would say that for now we change to C99.

If we find a way to replace the C++ comments from the stb_image.h, then we would do that later.